setup.sh

```
@@ -0,0 +1,95 @@
+sed -i "s/replace(\/\\\\\.git\\\\$\/, '')/endsWith('.git') ? headCloneUrl.slice(0, -4) : headCloneUrl/g" src/sessionContextMenu.ts
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)